local/index.js

```diff
@@ -132,6 +133,13 @@ function renderBlock(data) {
     ReactDOM.render(block, root);
 }
 
+// If tenant has been updated, set storeInformation
+if (!/\$/.test(clientUtils.client.tenant)) {
```

Comment:
Although nothing more than the `$` character is necessary, it might be helpful for readability to use the full `$YOUR_TENANT_ID` string here.

Reply:
23818f9e5e7123e2445631687eb563093806edf3